main.js

```diff
@@ -0,0 +1,64 @@
+const customerContainer = document.querySelector("#customers");
+// grab the customer container
+
+// console.log(customerContainer);
+
+// loop through the customer objects and build html for each one
+for (let customer of customers) {
+  // create a card for the customer
+  let card = document.createElement("div");
+  card.classList.add("card");
+  card.classList.add("small");
+
+  let name = document.createElement("div");
+  // create a div for name and apply the materialize class to it
+  name.classList.add("card-title");
+  let text = document.createTextNode(
+    `${titleCase(customer.name.first)} ${titleCase(customer.name.last)}`
+  );
+  name.classList.add("name");
+  name.appendChild(text);
+  // add the name to the card
+
+  // begin own work
+  let thumb = document.createElement("div");
```

Comment:
You could encapsulate each of these sections, which you have already organized, in its own function, too.